feature-libs/order/components/order-confirmation/order-confirmation-shipping/order-confirmation-shipping.component.html

```diff
@@ -1,12 +1,39 @@
 <div class="cx-order-items container" *ngIf="order$ | async as order">
-  <ng-template
-    [cxOutlet]="cartOutlets.DELIVERY_ITEM_LIST"
-    [cxOutletContext]="{
-      entries: entries,
-      readonly: true,
-      deliveryAddress: order.deliveryAddress,
-      deliveryMode: order.deliveryMode
-    }"
-  >
-  </ng-template>
+  <ng-container *ngIf="!showItemList || entries.length > 0">
```

Comment:
i want to confirm this logic makes sense?

we want to show this only when showItemFlag is false or there are entries?

Reply:
if `showItemList` is `true`, then need make sure `entries.length >0`. If not `showItemList` (only display address and delivery mode, e.g. for consignment), no need check `entries` (because entries are not used).